README.rst

```diff
@@ -109,8 +109,8 @@ Composer:
 
     {
        "require-dev": {
-          "ongr/ongr-strict-standard": "~1.0",
-          "squizlabs/php_codesniffer": "~1"
+          "ongr/ongr-strict-standard": "~2",
```

Comment:
From composer docs:
Note: The ~ operator has an exception on its behavior for the major release number. This means for example that ~1 is the same as ~1.0 as it will not allow the major number to increase trying to keep backwards compatibility.